source/common/quic/envoy_quic_proof_verifier.cc

```diff
@@ -63,6 +65,7 @@ class QuicValidateResultCallback : public Ssl::ValidateResultCallback {
   const std::string hostname_;
   // Leaf cert needs to be retained in case of asynchronous validation.
   std::string leaf_cert_;
+  const bool accept_untrusted_;
```

Comment:
nit: might be worth a comment, perhaps mentioning that this should really only be true in tests?

Reply:
in .h flile